mule-extensions-api/src/test/java/org/mule/runtime/extension/api/NameUtilsTestCase.java

```diff
@@ -91,6 +92,14 @@ public void hyphenization() {
     assertThat("parameter-dsl-name", is(hyphenize("ParameterDSLName")));
   }
 
+  @Test
+  public void defaultNamespaceWithCamelCaseWordsSeparatedBySpaces() {
+    assertThat(defaultNamespace("AbCd Properties Provider"), is("abcd-properties-provider"));
+    assertThat(defaultNamespace("AbCd properties provider"), is("abcdpropertiesprovider"));
```

Comment:
this assertion is incorrect. result should be `abcd-properties-provider`

Reply:
Hi @marianogonzalez! I thought the same as you, but according to this [test](https://jenkins-onprem.build.msap.io/job/Mule-runtime/job/Mule-4.x-Validation/job/master/21863/testReport/org.mule.runtime.extension.internal.loader/ModuleExtensionModelJsonTestCase/Parallel___Run_Tests___Mule___generate_Capitalized_with_spaces_in_the_name_json_/) that failed in a previous build, the expected result is what I put.